3529/2/public/313528695:313528695/checkins/07:24:2019:05/gannagood.txt

```diff
@@ -0,0 +1 @@
+1234567
```

Comment:
why I can see this? `.gitignore` for public folder
And I think you have a bad structure of folders `rubizza_number`/`checkins/checkout`/`timestamp`/
That's what we need, cos it's rly hard to look at time spent for some person without your matching table